aten/src/ATen/native/native_functions.yaml

```diff
@@ -4852,6 +4856,7 @@
   tags: inplace_view
   dispatch:
     CompositeExplicitAutograd: transpose_
+    NestedTensorCPU, NestedTensorCUDA: transpose_nested_
```

Comment:
suggest leaving in-place ops to the future if we decide we need that optimization